### PR TITLE
chromium-nosync@125.0.6422.61-r1287751: fix checkver regex not working + remove redundant 32bit version

### LINKED ISSUE #1749: [Bug]: Chromium-NoSync not updating to version 125
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

chromium-nosync

### Expected/Current Behaviour

Currently, the latest version of Chromium NoSync is v125.0.6422.61-r1287751, but due to a problem with the checkver regex, it stays on 124.0.6367.61-r1274542, presumably do to a change in the Github Releases page HTML.

### Steps to Reproduce

```console
PS> scoop search chromium-nosync
Results from local buckets...

Name            Version                Source   Binaries
----            -------                ------   --------
chromium-nosync 124.0.6367.61-r1274542 versions
```

### Possible Solution

Modify the checkver regex to fix the auto-update process

### Scoop and Buckets Version

```console
PS> scoop --version
Current Scoop version:
d337bb1f (HEAD -> master, tag: v0.4.2, origin/master, origin/HEAD) chore(release): Bump to version 0.4.2 (#5964)

'main' bucket:
e81b0e34d (HEAD -> master, origin/master, origin/HEAD) mihomo: Update to version 1.18.5

'extras' bucket:
0e8dfb384 (HEAD -> master, origin/master, origin/HEAD) swyh-rs: Update to version 1.10.6

'versions' bucket:
0fabfa1c6 (HEAD -> master, origin/master, origin/HEAD) prismlauncher-git: Update to version 7.0-3887cfe

'nerd-fonts' bucket:
2c010185 (HEAD -> master, origin/master, origin/HEAD) SarasaGothic: Update to version 1.0.12

'java' bucket:
b37d4acc (HEAD -> master, origin/master, origin/HEAD) liberica11-lite-jdk: Update to version 11.0.23-12
```

### Scoop Config

```console
PS> scoop config

last_update         use_external_7zip scoop_repo                              scoop_branch
-----------         ----------------- ----------                              ------------
18/05/2024 16:54:39              True https://github.com/ScoopInstaller/Scoop master
```

### PowerShell Version

```console
PS> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.4.2
PSEdition                      Core
GitCommitId                    7.4.2
OS                             Microsoft Windows 10.0.22631
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Additional Softwares

_No response_

### PR DESCRIPTION
**Pull request by NealVanMeerbeekPXL**
Closes #1749 

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by niheaven** [comment]
Put `url` and `hash` under architecture/64bit

**Comment by NealVanMeerbeekPXL** [comment]
> Put `url` and `hash` under architecture/64bit

I misunderstood that the single architecture manifest is only for 32bit

File: bucket/chromium-nosync.json [hunk 1]
```diff
@@ -38,22 +34,15 @@
     "persist": "User Data",
     "checkver": {
         "url": "https://github.com/Hibbiki/chromium-win64/releases/latest",
-        "regex": ">v([\\d.\\-r]+)</h1>"
+        "regex": ">v([.*-r]+)</h1>"
```
Comment (niheaven) [inline bucket/chromium-nosync.json:37]:
```suggestion
        "regex": ">v([\\d.]+-r\\d+)</h1>"
```

**Pull request merged**